.ci/ansible/Containerfile.j2

```diff
@@ -21,6 +21,9 @@ RUN pip3 install \
 {%- if item.name == "pulp-certguard" -%}
 {{ " " }}python-dateutil rhsm
 {%- endif -%}
+{%- if item.name == "pulp-2to3-migration" -%}
```

Comment:
if it doesn't work, try:
{%- if item.name == "pulpcore" -%}

Reply:
Container building part was too late but you set me to look into the right direction, thanks!